systemd/ceph-disk@.service

```diff
@@ -1,5 +1,7 @@
 [Unit]
 Description=Ceph disk activation: %f
+After=local-fs.target
+Wants=local-fs.target
```

Comment:
Minor nitpick, but shouldn't this be Requires=?
You want this unit to fail if the local fs can't be mounted (although, if that happens, you have bigger problems).